test/functional/utils/wake-db-up.js

```diff
@@ -1,33 +1,29 @@
-const typeorm = require('typeorm')
+const typeorm = require("typeorm");
```

Comment:
Please follow the same code style in this file as in all of the project: single quotes, no semicolon, dangling comma

Reply:
This is because the linter is excluding js files. Will fix this.